### DISCUSSION TITLE
Starting via Gitpod, nothing happens

### DISCUSSION THREAD

**Discussion by chivas03** [benzino77/tasmocompiler#323] [Q&A]
Dear All,

Could somebody maybe give me advice on how to fix my problem?
I start tasmocompiler via gitpod and the workspace loads and is visible in the gitpod workspace overview, but the browser just shows a white screen. First the message appears that connection will be established and then nothing happens anymore. I tried with Chrome, Edge and Firefox (on Windows 11 platform). I allowed all pop-ups to be shown - but nothing appears.

I tried it as well with my user profile on another laptop and there it works.

Has somebody faced the same issue or knows a workaround or if there is a log file available?

Regards,
Marco

**Comment by benzino77** [comment]
Try to open gitpod using "Private window" or "clear cookies" for gitpod site.

**Comment by chivas03** [comment]
I tried both, and nothing changed the result

**Comment by benzino77** [comment]
Anty virus/malware/etc. software? Firewall blocking websocket connections?

**Reply by chivas03** [reply]
No, normally nothing activated nor any information in a protocol about something has been blocked. However, I will check again.
